backend-shared/db/app_project_managed_env_test.go

```diff
@@ -66,4 +76,86 @@ var _ = Describe("AppProjectManagedEnvironment Test", func() {
 
 	})
 
+	Context("Test CountAppProjectManagedEnvironmentByClusterUserID function", func() {
```

Comment:
Not sure if this is much value other than to pad the coverage stats, I think we could cover the case

```
if err != nil {!
   return fmt.Errorf("unable to retrieve appProjectManagedEnvs with clusteruser_id: %v", err)
}
``` 
with something like (the main difference is the context):

```
		Context("Test error handling in ListAppProjectManagedEnvironmentByClusterUserId function", func() {
			It("should return no AppProjectManagedEnvironment", func() {
				_, managedEnvironment, _, _, _, err := db.CreateSampleData(dbq)
				Expect(err).ToNot(HaveOccurred())

				var clusterUser = &db.ClusterUser{
					Clusteruser_id: "test-user-application",
					User_name:      "test-user-application",
				}
				err = dbq.CreateClusterUser(ctx, clusterUser)
				Expect(err).ToNot(HaveOccurred())

				By("Verify whether AppProjectManagedEnvironment is created")
				appProjectManagedEnv := db.AppProjectManagedEnvironment{
					Clusteruser_id:         clusterUser.Clusteruser_id,
					Managed_environment_id: managedEnvironment.Managedenvironment_id,
					SeqID:                  int64(seq),
				}

				err = dbq.CreateAppProjectManagedEnvironment(ctx, &appProjectManagedEnv)
				Expect(err).ToNot(HaveOccurred())

				var appProjectManagedEnvs []db.AppProjectManagedEnvironment
				deadlineExceededContext, _ := context.WithTimeout(ctx, 1*time.Nanosecond)
				err = dbq.ListAppProjectManagedEnvironmentByClusterUserId(deadlineExceededContext, clusterUser.Clusteruser_id, &appProjectManagedEnvs)
				Expect(err).ToNot(HaveOccurred())
				Expect(appProjectManagedEnvs).To(HaveLen(0))
				Expect(appProjectManagedEnvs[0]).To(Equal(appProjectManagedEnv))

			})
		})

```